{"Sample.js":[{"testName":"Test fail","path":"out\\Sample.js","message":"{\"generatedMessage\":false,\"name\":\"AssertionError [ERR_ASSERTION]\",\"code\":\"ERR_ASSERTION\",\"actual\":false,\"expected\":true,\"operator\":\"==\",\"stackMessage\":\"AssertionError [ERR_ASSERTION]: One should be more than two!\\n    at TestCase.True (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Test\\\\Test.js:54:16)\\n    at Test_1.Test (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Sample.js:8:10)\\n    at Object.Test (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Test\\\\Test.js:69:9)\\n    at Object.<anonymous> (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Sample.js:7:8)\\n    at Module._compile (module.js:635:30)\\n    at Object.Module._extensions..js (module.js:646:10)\\n    at Module.load (module.js:554:32)\\n    at tryModuleLoad (module.js:497:12)\\n    at Function.Module._load (module.js:489:3)\\n    at Function.Module.runMain (module.js:676:10)\",\"errorMessage\":\"One should be more than two!\"}","passed":false}],"Example.js":[{"testName":"Test valid name","path":"out\\Example.js","message":"OK","passed":true},{"testName":"Test fail","path":"out\\Example.js","message":"{\"generatedMessage\":false,\"name\":\"AssertionError [ERR_ASSERTION]\",\"code\":\"ERR_ASSERTION\",\"actual\":false,\"expected\":true,\"operator\":\"==\",\"stackMessage\":\"AssertionError [ERR_ASSERTION]: One should be more than two!\\n    at TestCase.True (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Test\\\\Test.js:54:16)\\n    at Test_1.Test (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Example.js:21:10)\\n    at Object.Test (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Test\\\\Test.js:69:9)\\n    at Object.<anonymous> (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Example.js:20:8)\\n    at Module._compile (module.js:635:30)\\n    at Object.Module._extensions..js (module.js:646:10)\\n    at Module.load (module.js:554:32)\\n    at tryModuleLoad (module.js:497:12)\\n    at Function.Module._load (module.js:489:3)\\n    at Function.Module.runMain (module.js:676:10)\",\"errorMessage\":\"One should be more than two!\"}","passed":false},{"testName":"Test changed name","path":"out\\Example.js","message":"OK","passed":true}],"One.js":[{"testName":"Test increment","path":"out\\One\\One.js","message":"{\"generatedMessage\":true,\"name\":\"AssertionError [ERR_ASSERTION]\",\"code\":\"ERR_ASSERTION\",\"actual\":65,\"expected\":64,\"operator\":\"===\",\"stackMessage\":\"AssertionError [ERR_ASSERTION]: 65 === 64\\n    at TestCase.StrictEquals (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Test\\\\Test.js:28:16)\\n    at Test_1.Test (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\One\\\\One.js:8:10)\\n    at Object.Test (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\Test\\\\Test.js:69:9)\\n    at Object.<anonymous> (C:\\\\Users\\\\admin\\\\Documents\\\\Personal Projects\\\\MyoTest\\\\out\\\\One\\\\One.js:4:8)\\n    at Module._compile (module.js:635:30)\\n    at Object.Module._extensions..js (module.js:646:10)\\n    at Module.load (module.js:554:32)\\n    at tryModuleLoad (module.js:497:12)\\n    at Function.Module._load (module.js:489:3)\\n    at Function.Module.runMain (module.js:676:10)\",\"errorMessage\":\"65 === 64\"}","passed":false}]}